nestkernel/event_delivery_manager.cpp

```diff
@@ -143,10 +145,13 @@ EventDeliveryManager::resize_send_recv_buffers_target_data()
 void
 EventDeliveryManager::resize_send_recv_buffers_spike_data_()
 {
-  send_buffer_spike_data_.resize( kernel().mpi_manager.get_buffer_size_spike_data() );
-  recv_buffer_spike_data_.resize( kernel().mpi_manager.get_buffer_size_spike_data() );
-  send_buffer_off_grid_spike_data_.resize( kernel().mpi_manager.get_buffer_size_spike_data() );
-  recv_buffer_off_grid_spike_data_.resize( kernel().mpi_manager.get_buffer_size_spike_data() );
+  if ( kernel().mpi_manager.get_buffer_size_spike_data() > send_buffer_off_grid_spike_data_.size() )
```

Comment:
i was a bit confused that you use the size of `off_grid_spike_data` here, maybe just use `send_buffer_spike_data`? it's not wrong, i just needed to check twice

Reply:
Yes, currently it makes no real difference which is used. That you checked twice was sort of the aim. My logic: if, in the future, the sizes of `send_buffer_off_grid_spike_data_` and `send_buffer_spike_data_` are no longer equal, this line will serve to prompt the developer that some attention is needed. It's a bit ugly, but I thought it added some value.